src/windows/views/blender_listview.py

```diff
@@ -843,9 +843,8 @@ def Render(self):
 
         _ = get_app()._tr
 
-        if not self.version:
-            if not self.blender_version_check():
-                return
+        if not self.version and not self.blender_version_check():
+            return
```

Comment:
Heh. I suspect I wrote this like that because (looking at it now) `self.blender_version_check()` can raise an `Exception`, something I should probably be handling here but am not. (Clearly, writing it "wrong" was a poor reminder, as it didn't stop me from merging it without addressing that. So a `TODO` or `XXX` comment would've been a better choice.)

Anyway, point is, this should really have a `try:` wrapped around the inner condition, so it may end up getting changed some more — but that's unrelated to **_this_** change. Plus, addressing the exception issue may involve having `blender_version_check()` just return `False` instead of raising the exception in the first place.

Point is, outside issues aside, for the current code combining them is the right call.